### DISCUSSION TITLE
🐛 [[firebase_crashlytics]] Support older version of Flutter.

### DISCUSSION THREAD

**Discussion by niranjan-b-prajapati** [firebase/flutterfire#5633] [Feature request]
## Bug report

**Describe the bug**
I am having a project which is on Flutter version 1.12.13+hotfix.9 and I am trying to integrate the firebase crashlytics but getting the below error while doing `pub get`:

```
Running "flutter pub get" in myApp...                       
The current Dart SDK version is 2.7.2.

Because myApp depends on firebase_crashlytics >=0.2.0-dev.1 which requires SDK version >=2.8.0 <3.0.0, version solving failed.


pub get failed (1; Because myApp depends on firebase_crashlytics >=0.2.0-dev.1 which requires SDK version >=2.8.0 <3.0.0, version solving failed.)
exit code 1
```

The `firebase_core: ^0.5.0` was installed and integrated successfully. Stuck with the firebase crashlytics 

### Expected behavior

I expected the Firebase crashlytics to support older versions of Flutter as currently we are not allowed to perform an upgrade on flutter version.

### Flutter doctor

Run `flutter doctor` and paste the output below:

<details><summary>Click To Expand</summary>

```
[✓] Flutter (Channel stable, v1.12.13+hotfix.9, on Mac OS X 10.15.6 19G2021, locale en-IN)
    • Flutter version 1.12.13+hotfix.9 at /Library/FlutterSdk/flutter
    • Framework revision f139b11009 (6 months ago), 2020-03-30 13:57:30 -0700
    • Engine revision af51afceb8
    • Dart version 2.7.2

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /usr/local/opt/android-sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.3
    • ANDROID_HOME = /usr/local/opt/android-sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.7)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.7, Build version 11E801a
    • CocoaPods version 1.9.1

[✓] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 47.1.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] VS Code (version 1.49.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.14.1

[✓] Connected device (1 available)
    • Mi A3 • 5380418c1454 • android-arm64 • Android 10 (API 29)

• No issues found!
```

</details>

---

### Flutter dependencies

Run `flutter pub deps -- --style=compact` and paste the output below:

<details><summary>Click To Expand</summary>

```
No .dart_tool/package_config.json file found, please run "pub get" first.

Starting with Dart 2.7 this file configures the resolution of package import URIs. Running "pub get" will generate this file.
pub finished with exit code 65
```

</details>

---
As we are nearing the delivery we need the crashlytics to be integrated in the app for better tracking and resolving crashes. Please help me get this issue resolved.

**Comment by darshankawar** [comment]
@niranjan-b-prajapati 
Since you are using very old Flutter verison (1.12.13+hotfix.9), not sure if the plugin supports backward compatibility.
Keeping this issue open for further analysis.

**Comment by niranjan-b-prajapati** [comment]
Thank you @darshankawar for acknowledging the issue,  Hope there is some backward compatibility support. As our app is nearing the delivery we are very keen on integrating the Firebase crashlytics. As the earlier versions of fabric support had issues in the integration. we have waited for the crashlytics to come to flutter. But unfortunately, it is not supported in flutter version 1.12.13+hotfix.9 

Hope someone helps support it. Else we will have to launch the app without crashlytics integrated on it.

**Comment by longravuth** [comment]
Same issue here

**Comment by akanksha820** [comment]
same issue here
